docs/configuration.md

```diff
@@ -626,19 +626,6 @@ Apart from these, the following properties are also available, and may be useful
     You can mitigate this issue by setting it to a lower value.
   </td>
 </tr>
-<tr>
-  <td><code>spark.maxRemoteBlockSizeFetchToMem</code></td>
-  <td>Int.MaxValue - 512</td>
```

Comment:
just to clarify, you intentionally moved this from shuffle section to network section  since it affects both the shuffle fetch and block manager fetches?

Reply:
yea